homeassistant/components/concord232/alarm_control_panel.py

```diff
@@ -79,7 +83,7 @@ class Concord232Alarm(alarm.AlarmControlPanelEntity):
         | AlarmControlPanelEntityFeature.ARM_AWAY
     )
 
-    def __init__(self, url, name, code, mode):
+    def __init__(self, url, name, code, mode) -> None:
```

Comment:
Please do not partial type. Either fully type the method or do not type it at all.